### PR TITLE
Thomas/feature/110

### PR DESCRIPTION
**Pull request by Unaimend**
Add the ability of the matchOnNames function to output the names that lead to this match into the output.tsv. Useful for debugging

### CODE REVIEW

File: src/MeMoModel.py [hunk 1]
```diff
@@ -81,11 +81,13 @@ def annotate(self) -> None:
         print("Total:", anno_result)
 
 
-    def match(self, model2: MeMoModel, keep1ToMany:bool = True) -> pd.DataFrame:
-        """ compares the metabolites of two models and returns a data frame with additional information """
+    def match(self, model2: MeMoModel, keep1ToMany:bool = True, output_names: bool = False) -> pd.DataFrame:
```
Comment (Unaimend) [inline src/MeMoModel.py:84]: IF https://github.com/Porthmeus/MeMoMe/pull/89 gets merged we have to add a parameter to the main program that enables/disables the output_names stuff

Comment (Unaimend) [reply src/MeMoModel.py:84]: #89 got merged. Add the additional parameter ASAP

Comment (Porthmeus) [reply src/MeMoModel.py:84]: please fix #89 and I will review

**Comment by Unaimend** [comment]
Check whats the difference with 108

**Comment by Unaimend** [comment]
Already merged through different branch

**Pull request closed without merging**